.env

```diff
@@ -2,8 +2,8 @@
 #VITE_BAHIS_SERVER_URL=http://www.sharful.com:3001
 #VITE_BAHIS_KOBOTOOLBOX_KF_API_URL=https://kf.kobotoolbox.org/api/v2/
 #VITE_BAHIS_KOBOTOOLBOX_KC_API_URL=https://kc.kobotoolbox.org/api/v1/
-#VITE_BAHIS_KOBOTOOLBOX_API_TOKEN=a8bad2ac778280d75d6c45868a68061ea8a81ba0
-
+##VITE_BAHIS_KOBOTOOLBOX_API_TOKEN=a8bad2ac778280d75d6c45868a68061ea8a81ba0
+#VITE_BAHIS_KOBOTOOLBOX_API_TOKEN=177aee1c33349cd3d7734b9a39e0da03a23e2e60
```

Comment:
This is a public repo - make sure not to publish working tokens here (use .env.local for those)